### PR TITLE
snapcraft: Updates snapcraft.yaml to build lxd-imagebuilder and simplestream-maintainer

### PR DESCRIPTION
**Pull request by tomponline**

### CODE REVIEW

File: snapcraft.yaml [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
+name: lxd-imagebuilder
 base: core22
 version: git
 grade: devel
-summary: System container and VM image builder for LXC and Incus
+summary: System container and VM image builder for LXD
 description: |-
-  Distrobuilder is a tool which lets you create container and VM images for LXC and Incus.
-license: Apache-2.0
+  LXD image builder is a tool which lets you create system container and VM images for use with LXD.
```
Comment (MusicDin) [inline snapcraft.yaml:7]: Maybe worth mentioning that the snap package also includes simplestream-maintainer binary

**Review by MusicDin — APPROVED** [review]
Other then description, looks good to me

**Comment by MusicDin** [comment]
Also, if possible, I think `xdelta3` should be included, as it is required by simplestream-maintainer. However, not a priority, as I would like to see lxd-imagebuilder out soon, so that we can get it as Snap in lxd-ci

**Comment by tomponline** [comment]
I'll  merge this for now and you can follow up with those changes, as im focused on getting the end-to-end build working right now.

As for the extra dep, im not sure if thats needed given its a classic snap, it maybe these need to be installed on the host itself.

**Pull request merged**